app/assets/stylesheets/game_index.scss

```diff
@@ -0,0 +1,22 @@
+body {
+  background-color: #0f253a !important;
+}
+
+ul {
+  padding: 0 !important;
```

Comment:
As @kellylougheed - `!important` is bad practice & cannot be overridden. CSS declaration order should be your determination of what styles override others. Read more - https://css-tricks.com/when-using-important-is-the-right-choice/